File: Difficulty 1.6/Joint Jog Jam/joint.cpp
#include <math.h>
#include <iostream>

using namespace std;

double dist(double a, double b, double c, double d) {
	double ans = 0;
	ans = sqrt(pow((a - c), 2) + pow((b - d), 2));

	return ans;
}

int main() {
	double kariStartX, kariStartY, olaStartX, olaStartY;
	double kariEndX, kariEndY, olaEndX, olaEndY;

	cin >> kariStartX;
	cin >> kariStartY;
	cin >> olaStartX;
	cin >> olaStartY;
	cin >> kariEndX;
	cin >> kariEndY;
	cin >> olaEndX;
	cin >> olaEndY;

	double ans = 0;

	double first = dist(kariStartX, kariStartY, olaStartX, olaStartY);
	if (first > ans) {
		ans = first;
	}

	double second = dist(kariEndX, kariEndY, olaEndX, olaEndY);
	if (second > ans) {
		ans = second;
	}

	cout << ans << endl;
	return 0;
}

File: Difficulty 1.6/Joint Jog Jam/joint.py
import math

a = list(map(int, input().strip().split()))

ans = 0

first = math.sqrt(pow(a[0] - a[2], 2) + pow(a[1] - a[3], 2))
second = math.sqrt(pow(a[4] - a[6], 2) + pow(a[5] - a[7], 2))

result = max(first, second)
print(result)
